.github/workflows/testing.yml

```diff
@@ -3,131 +3,84 @@ name: Pytest MSS
 on:
   workflow_call:
     inputs:
-      xdist:
-        required: true
-        type: string
-      branch_name:
-        required: true
-        type: string
-      event_name:
+      image_suffix:
         required: true
         type: string
     secrets:
       PAT:
-        required: true
 
 env:
-  PAT: ${{ secrets.PAT }}
-  EVENT: ${{ inputs.event_name }}
+  mamba-env: mss-${{ inputs.image_suffix }}-env
 
 jobs:
   Test-MSS:
     runs-on: ubuntu-latest
 
-    defaults:
-      run:
-        shell: bash
-
     container:
-      image: openmss/testing-${{ inputs.branch_name }}
+      image: openmss/testing-${{ inputs.image_suffix }}
 
-    steps:
-    - name: Trust My Directory
-      run: git config --global --add safe.directory /__w/MSS/MSS
+    strategy:
+      fail-fast: false
+      matrix:
+        order: ["normal", "reverse"]
 
+    steps:
     - uses: actions/checkout@v4
 
     - name: Check for changed dependencies
-      run: |
-        cmp -s /meta.yaml localbuild/meta.yaml && cmp -s /development.txt requirements.d/development.txt \
-        || (echo Dependencies differ \
-            && echo "triggerdockerbuild=yes" >> $GITHUB_ENV )
+      run: cmp -s /meta.yaml localbuild/meta.yaml && cmp -s /development.txt requirements.d/development.txt ||
+        (echo Dependencies differ && echo "triggerdockerbuild=yes" >> $GITHUB_ENV )
 
     - name: Install pyvirtualdisplay if on stable
-      if: ${{ inputs.branch_name == 'stable' }}
+      if: ${{ inputs.image_suffix == 'stable' }}
       run: |
         source /opt/conda/etc/profile.d/conda.sh
         source /opt/conda/etc/profile.d/mamba.sh
-        mamba install -n mss-${{ inputs.branch_name }}-env pyvirtualdisplay
+        mamba install -n mss-${{ inputs.image_suffix }}-env pyvirtualdisplay
```

Comment:
Spoke to soon, since CI treats stable and develop differently. This entire step has to be removed.

Reply:
ok, we can do that for now, but then we should also drop pyvirtualdisplay in the new stable asap.

Reply:
What you did isn't what I meant. The entire block I've commented on should be removed, because pyvirtualdisplay is no longer needed on stable after the merge.

`inputs.branch_name` doesn't exist, so your change has the same effect as removing the entire step (it gets skipped), but it should be fully removed.

Reply:
done